BPSampleApp/BPSampleAppCrashingTests/BPSampleAppCrashingTests.m

```diff
@@ -15,21 +15,6 @@ @interface BPSampleAppCrashingTests : XCTestCase
 
 @implementation BPSampleAppCrashingTests
 
-NSString *attemptFromSimulatorVersionInfo(NSString *simulatorVersionInfo) {
```

Comment:
Let's keep this and the tests, just disable them.

Reply:
Done.